pkg/cmd/api/api_test.go

```diff
@@ -773,11 +770,6 @@ func Test_apiRun_cache(t *testing.T) {
 		CacheTTL:    time.Minute,
 	}
 
-	t.Cleanup(func() {
-		cacheDir := filepath.Join(os.TempDir(), "gh-cli-cache")
-		os.RemoveAll(cacheDir)
-	})
```

Comment:
This one wasn't even used, since: https://github.com/cli/cli/blob/25d79c4e16d54d6f3621517269f07048748a6b97/pkg/cmd/api/api.go#L268-L270
already creates a temporal directory for caching.

Reply:
This was hacky, but was absolutely used: `NewCachedClient()` uses this location on disk, so tests need to clear it between runs.